.docker/app/Dockerfile

```diff
@@ -1,12 +1,26 @@
-FROM ruby:2.6
-RUN \
-      wget -qO- https://deb.nodesource.com/setup_9.x | bash - && \
-      wget -qO- https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
-      echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
-      apt-get update -qq && \
-      apt-get install -y --force-yes --no-install-recommends && \
-      nodejs yarn build-essential libpq-dev \
-      && apt-get clean
-RUN gem install bundler
+ARG ALPINE_RUBY_VERSION
```

Comment:
I'm using `ALPINE_RUBY_VERSION` because `RUBY_VERSION` is already in use elsewhere, e.g., by RVM. This allows switching the version of Ruby used in the app image via environment variable:

```shell
$ ALPINE_RUBY_VERSION=2.7 docker-compose up --build app
```